### ISSUE TITLE
webpack aliased import

### ISSUE THREAD

**Issue by bsr203** [Drathal/react-playground#2]
Hi Markus,

While looking to speedup my unit tests, I found your repo as a good example of jsdom + webpack + enzyme setup. While trying out, I have one issue though. I use webpack `resolve.alias` not only to avoid relative paths, but also mock some imports during testing. I found it doesn't work with the current setup. I see an issue reported here https://github.com/istarkov/babel-plugin-webpack-loaders/issues/68

Do you think it is possible to use all webpack configuration and support aliased imports.

thanks a lot for open sourcing this package.

cheers,
bsr.

**Comment by Drathal** [comment]
Hi @bsr203 

Thanks for reporting that. :)
I will play around with `resolve.alias ` and will post my outcome here.
Maybe you can post an example how you use it .

Cheers
Markus

**Comment by Drathal** [comment]
@bsr203  pushed a new version  (15ddf6d5565b0130356cecae9534b61b10f39249) with example usage of `resolve.alias`. Its a very simple usecase but it seems to work for me.

**Comment by bsr203** [comment]
Hi Markus,

Thanks a ton for your help and effort. It work splendid now, as the running time for a test reduce to 1 sec from about 20!. I will outline some of the issues I faced, if anyone experience the same.

1. multiple .babelrc files.

the plugins for testing are configured at top level and I has some other `.babel.rc` files in the child directories. It took me a while to figure this out as the cause of `babel-plugin-webpack-aliases` is not loaded.

2. babel-plugin-webpack-aliases vs babel-plugin-webpack-alias

Initially I tried with `babel-plugin-webpack-alias` and I only later noticed you used `babel-plugin-webpack-aliases`, which is a fork of the former. I was debugging the wrong plugin to see why aliases are not overwritten. Former didn't work because of https://github.com/trayio/babel-plugin-webpack-alias/pull/10, and the patch made it work. `babel-plugin-webpack-aliases` already includes it, so works out of the box.

3. I was using `require.context` to dynamically load modules. I mocked it with the implementation from [here](https://github.com/halt-hammerzeit/webpack-isomorphic-tools/blob/master/source/index.js#L321) for node builds.

thanks again for your help.

cheers.
bsr.
